plugins/modules/gitlab_group.py

```diff
@@ -255,7 +255,10 @@ def create_group(self, arguments):
             return True
 
         try:
-            group = self._gitlab.groups.create(arguments)
+            # Filter out None values
+            filtered = {arg_key: arg_value for arg_key, arg_value in arguments.items() if arg_value is not None}
```

Comment:
Needed for Python 2.6:
```suggestion
            filtered = dict(arg_key=arg_value for arg_key, arg_value in arguments.items() if arg_value is not None)
```